### PR TITLE
Bug 1279796 - (akide-sidebar-add-page) add a page from the akita sidebar

### PR DESCRIPTION
**Pull request by mancas**

### CODE REVIEW

**Comment by dmose** [comment]
@mancas tests are broken

**Comment by mancas** [comment]
@dmose Thanks! Fixed now

File: shared/js/textChatView.jsx [hunk 1]
```diff
@@ -387,11 +394,59 @@ loop.shared.views.chat = (function(mozL10n) {
...
+      });
+
+      // XXX akita: this is a workaround till Bug 1273497 is landed
+      loop.request("GetSelectedTabMetadata").then(metadata => {
+        loop.shared.utils.getPageMetadata(metadata.url).then(result => {
+          this.setState({
+            success: true,
+            working: false
+          });
+          // Reset state after 2 seconds
```
Comment (daveccrit) [inline shared/js/textChatView.jsx]: 2 or 3 seconds?

File: shared/js/textChatView.jsx [hunk 2]
```diff
@@ -387,11 +394,59 @@ loop.shared.views.chat = (function(mozL10n) {
...
+          setTimeout(this._resetState, this.constructor.RESET_TIMEOUT);
+          this.props.dispatcher.dispatch(new sharedActions.AddPage(result));
+        }).catch(() => {
+          this.setState({
+            error: true,
+            working: false
+          });
+          // Reset state after 2 seconds
```
Comment (daveccrit) [inline shared/js/textChatView.jsx]: same here

**Comment by daveccrit** [comment]
@mancas Getting 4 failures in the UI showcase under Local Tests

**Comment by mancas** [comment]
@daveccrit fixed =)

**Comment by daveccrit** [comment]
r=dcritch

**Pull request merged**